practice_05.11.19_classes/Task2_Plants/Worker.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+
+namespace practice_05._11._19_classes.Task2_Plants
+{
+    public class Worker
+    {
+        public void ManagePlant(Plant plant)
+        {
```

Comment:
Please don't forget to add validation for all public members to check if parameters are not null and valid.

Reply:
fixed